src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -98,35 +88,43 @@ abstract class ManagedStrategy(
     // used on resumption, and the trace point before and after the suspension
     // correspond to the same method call in the trace.
     private val suspendedFunctionsStack = Array(nThreads) { mutableListOf<Int>() }
+    // Helps to ignore potential switch point in local objects (see LocalObjectManager) to avoid
+    // useless interleavings analysis
```

Comment:
In the project, we always put "." at the end of each sentence in the documentation. Please follow this style and check other comments.